integration-auth-api-client/src/main/java/org/symphonyoss/integration/auth/api/client/KmAuthHttpApiClient.java

```diff
@@ -46,7 +53,7 @@ protected String getBasePath() {
     String url = properties.getKeyManagerAuthUrl();
 
     if (StringUtils.isBlank(url)) {
-      throw new MissingConfigurationException(SERVICE_NAME, REQUIRED_KEY);
+      throw new MissingConfigurationException(SERVICE_NAME, REQUIRED_KEY, logMessageSource.getMessage(KEY_MANAGER_URL_SOLUTION, REQUIRED_KEY));
```

Comment:
You have just provided the solution. You should provide the message content as well.

Something like that:
```
String message = logMessageSource.getMessage(MISSING_CONFIG_INFO, REQUIRED_KEY);
String solution = logMessageSource.getMessage(MISSING_CONFIG_INFO_SOLUTION, REQUIRED_KEY);
throw new MissingConfigurationException(SERVICE_NAME, message, solution);
```